intellij-plugin-verifier/verifier-intellij/src/main/java/com/jetbrains/pluginverifier/verifiers/PluginVerificationContext.kt

```diff
@@ -103,11 +102,7 @@ data class PluginVerificationContext(
   }
 
   override fun registerDeprecatedUsage(deprecatedApiUsage: DeprecatedApiUsage) {
-    val deprecatedElementHost = deprecatedApiUsage.apiElement.containingClass
-    val usageHostClass = deprecatedApiUsage.usageLocation.containingClass
-    if (deprecatedApiUsage is DiscouragingJdkClassUsage || shouldIndexDeprecatedClass(usageHostClass, deprecatedElementHost)) {
-      deprecatedUsages += deprecatedApiUsage
-    }
+    deprecatedUsages += deprecatedApiUsage
   }
 
   override fun registerExperimentalApiUsage(experimentalApiUsage: ExperimentalApiUsage) {
```

Comment:
Can't write comment for the code below for some reason. The question for `if (shouldIndexDeprecatedClass(usageHostClass, elementHostClass))` -- shouldn't it be removed as well?